### ISSUE TITLE
Can you please share the parameters values used for Linux data in the blogpost

### ISSUE THREAD

**Issue by mehanelson** [karpathy/char-rnn#173]
In the RNN blogpost, the model is tested on Linux Source Code with input.txt file of 474MB of C code. @karpathy Can you please share what were the parameters values you used for training it (e.g. sequence length, batch size, learning rate, rnn size, dropout, decay rate etc. )
 Thank you!

**Comment by yashmaverick** [comment]
@mehanelson  Can you please share the parameters values in case if you have successfully trained the model.

**Comment by brucecui1998** [comment]
Could you please share the Linux kernel data file? I cound not access the [stanford website](https://www.cs.stanford.edu/people/karpathy/char-rnn/)
